go-controller/pkg/ovn/secondary_layer3_network_controller.go

```diff
@@ -453,7 +570,11 @@ func (oc *SecondaryLayer3NetworkController) syncNodes(nodes []interface{}) error
 			continue
 		}
 
-		foundNodes.Insert(node.Name)
+		// Add the node to the foundNodes only if it belongs to the local zone.
```

Comment:
Don't we need to call the IC handler syncNodes here?

Reply:
Yup.  we need to.  Thanks for finding this.